### PR TITLE
End user boards deprecation notice

### PR DESCRIPTION
**Pull request by esethna**
Add end user notice for boards deprecation

**For testing, this should only appear for cloud instances**

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -75,8 +75,23 @@
...
+        "title": "Mattermost Boards will be removed on September 28th, 2023",
+        "description": "Mattermost Boards is transitioning to a fully community supported plugin. As a result, Boards will be removed from Mattermost Cloud instances on September 28th, 2023. Click below to learn more about how this impacts your instance and how to prepare.",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Feedback.png",
+        "actionText": "Learn more",
+        "actionParam": "https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669"
+      }
+    }
+  },
+  {
+    "id": "boards_deprecations_end_user",
```
Comment (amyblais) [inline notices.json:87]: @esethna I believe the build is failing because the max character length for id's is 26. The id here is too long.

Comment (esethna) [reply notices.json]: Thanks, good catch, updated

**Comment by lindy65** [comment]
As I haven't tested notices before, Linda is going to give me some training on Tuesday (Monday is a holiday in the US/Canada). Yasser has kindly agreed to test this PR in the meantime - thanks Yasser 🙌

**Review by yasserfaraazkhan — APPROVED** [review]
tested on latest 9.0.0-rc1
system admin
Screenshot 2023-09-02 at 4 26 31 PM

user
image

**Pull request merged**